Core/UserDrivers/ADCDriver.c

```diff
@@ -26,3 +26,16 @@ PUBLIC result_t ADCGetThrottleRaw(throttle_raw_t * throttleRaw)
 
 	return RESULT_OK;
 }
+
+PUBLIC percentage_t ADCConvertThrottle(throttle_raw_t *throttleRaw) {
+
+	if (*throttleRaw > ADC_MAX_THROTTLE) {
+		return 1000;
+	}
+
+	if (*throttleRaw <= ADC_MIN_THROTTLE) {
+		return 0;
+	}
+
+	return (*throttleRaw - ADC_MIN_THROTTLE) * ADC_MAX_PERCENTAGE / ADC_MAX_THROTTLE;
```

Comment:
Looks like percent is an integer value, so max would be 100. do you need a 16 bit int for that?

Reply:
Not sure what the best way to represent percentage is for STM. I'd like to have up to 0.1% precision. Thus ranging from 0 to 1000 gives 0% to 1000%